source/edge-server/configure.txt

```diff
@@ -134,271 +115,162 @@ EC2, you must open ports to accept incoming connections:
 - Development: Use HTTP on port 80 for simplicity and speed of development.
 - Production: Use SSL with port 443 to secure access to production data.
 
-Edge Server supports TLS. You can find configuration details in the 
-"Complete the Edge Server Configuration Details" section on this page.
+Edge Server supports TLS.
 
-.. _edge-server-os-support:
-
-OS Compatibility
-~~~~~~~~~~~~~~~~
-
-The Edge Server has been verified with the following operating systems:
-
-- Ubuntu Server 18+
-- macOS 11.x and newer running on M1 or Intel 2018+ MacBook Pros
-
-Other operating systems may be possible, but have not yet been verified.
-
-.. _edge-server-hardware-requirements:
-
-Hardware Requirements
-~~~~~~~~~~~~~~~~~~~~~
-
-Edge Server benchmarking is not yet complete, so minimum required hardware
-is not yet available. 
+[PLACEHOLDER: Add a link to details about configuring TLS once engineering
```

Comment:
not sure if rushil's already talked to you about this, but TLDR the user is responsible for TLS termination for HTTP (including device SDK traffic), which we'd expect them to set up via a kubernetes controller.

for wireprotocol, the user will set it up via edgectl, more info coming soon

Reply:
Nope, I haven't gotten those details from Rushil yet. Since we don't have customers doing this yet, I'll add some placeholder language around this. We can build out more details if needed when customers get to this point.